### ISSUE TITLE
ag-paging-row-summary-panel ag-9-last-row not updating

### ISSUE THREAD

**Issue by jasonparallel** [ag-grid/ag-grid#4353]
```
[X] bug report => see 'Providing a Reproducible Scenario'
[] feature request => do not use Github for feature requests, see 'Customers of ag-Grid'
[] support request => see 'Requesting Community Support'
```

Issue visible on demo page https://www.ag-grid.com/documentation/angular/infinite-scrolling/#example-1-auto-pagination-page-size-large-infinite-block-size
"Example 1: Auto Pagination Page Size, Large Infinite Block Size" & "Example 2: Equal Pagination Page Size and Large Infinite Block Size"

When the data first loads the ag-paging-row-summary-pane area update to the text "1 to ? of more". If you go one page forward and then back to the fist page it will render the correct "1 to 10 of more". This issue is seem to happen every time the row data passes over a block and data from a new block is fetched.

**Comment by jasonparallel** [comment]
Some quick debugging shows that endRow is set correctly 
 if (this.rowNodeBlockLoader.isLoading()) {
prevents it from being rendered 
isLoading is cleared by the loadComplete event
But onPaginationChanged is never called again after  the loadComplete event

**Comment by jasonparallel** [comment]
Perhaps BLOCK_LOADER_FINISHED_EVENT should trigger a paginationChanged

**Comment by makinggoodsoftware** [comment]
Hi,

Note that this was fixed in v25.1, released a couple of days ago

Thanks for your feedback!
